lib/lm_docstache/hide_custom_tags.rb

```diff
@@ -29,6 +29,7 @@ def hide_custom_tags!
           run_nodes = paragraph.css('w|r')
           while run_node = run_nodes.shift
             next if run_node.text.to_s.strip.size == 0
+            next unless run_node.at_css('w|t')
```

Comment:
Weird that you had to appeal to adding this line. The line above it should do the trick. Interesting...